README.md

```diff
@@ -59,37 +49,37 @@ TODO: Add other package managers:
 
 ### Released Binaries
 
-We provide [static-compiled binaries](https://github.com/scaleway/scaleway-cli/releases/latest) for darwin (macOS), GNU/Linux, and Windows platforms.
-You just have to download the binary compatible with your platform to a directory available in your `PATH`:
+We provide [static-compiled binaries](https://github.com/scaleway/scaleway-cli/releases/latest) for [darwin (macOS)](#mac-os), [GNU/Linux](#linux), and [Windows](#windows) platforms.
+You just have to download the binary compatible with your platform to a directory available in your [`PATH`](https://en.wikipedia.org/wiki/PATH_(variable)):
 
 #### Mac OS
 
 ```bash
 # Check that /usr/local/bin is in your PATH
-echo $PATH
+$ echo $PATH
 /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 
 # Download the release from github
-curl -o /usr/local/bin/scw -L "https://github.com/scaleway/scaleway-cli/releases/download/v2.0.0-beta.2/scw-2-0-0-beta-2-darwin-x86_64"
+$ curl -o /usr/local/bin/scw -L "https://github.com/scaleway/scaleway-cli/releases/download/v2.0.0-beta.2/scw-2-0-0-beta-2-darwin-x86_64"
 
 # Allow executing file as program
-chmod +x /usr/local/bin/scw
+$ chmod +x /usr/local/bin/scw
 
 # Init the CLI
-scw init
+$ scw init
 ```
 
 #### Linux
 
 ```bash
 # Download the release from github
-sudo curl -o /usr/local/bin/scw -L "https://github.com/scaleway/scaleway-cli/releases/download/v2.0.0-beta.2/scw-2-0-0-beta-2-linux-x86_64"
+$ sudo curl -o /usr/local/bin/scw -L "https://github.com/scaleway/scaleway-cli/releases/download/v2.0.0-beta.2/scw-2-0-0-beta-2-linux-x86_64"
```

Comment:
Not in favor of adding the $ sign because it makes it impossible to copy and paste the link and have it work